cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -411,6 +411,24 @@ public void getAll(@NotNull Context ctx) {
             String begin = ctx.queryParam(BEGIN);
             String end = ctx.queryParam(END);
             String timezone = ctx.queryParamAsClass(TIMEZONE, String.class).getOrDefault("UTC");
+            String versionDateParam = ctx.queryParam(VERSION_DATE);
+            ZonedDateTime versionDate = null;
+            if (versionDateParam != null) {
+                versionDate = DateUtils.parseUserDate(versionDateParam, timezone);
+            }
+
+            String versionTypeParam = ctx.queryParam(VERSION_TYPE);
+            VersionType versionType = VersionType.versionTypeFor(versionTypeParam);
+
+            // Throw error if max aggregate version type is requested with a version date
+            // or single version version type is requested without a version date
+            if(Objects.equals(versionType, VersionType.MAX_AGGREGATE) && versionDate != null) {
```

Comment:
no need for `Objects.equals`, enums can be `==` and that's easier to read

Reply:
done